index.js

```diff
@@ -20,21 +22,34 @@ const input = {
  * @param {string} action
  * @param {Record<string, unknown>} params
  */
-function callAliyunApi(endpoint, apiVersion, action, params) {
-  return new AliyunClient({
-    ...input.accessKeyId && input.accessKeySecret ? {
-      accessKeyId: input.accessKeyId,
-      accessKeySecret: input.accessKeySecret
-    } : {},
-    ...input.securityToken ? {
-      securityToken: input.securityToken
-    } : {},
-    endpoint,
-    apiVersion
-  }).request(action, params, { method: "POST" });
+function callAliyunApi (endpoint, apiVersion, action, params) {
```

Comment:
Please remove all the extra space between function name and left square bracket.

Reply:
done